pkg/addon-operator/operator.go

```diff
@@ -2136,6 +2163,16 @@ func (op *AddonOperator) CreateConvergeModulesTasks(state *module_manager.Module
 			doModuleStartup = true
 		}
 
+		// create ensureCRDsTasks
+		ensureCRDsTasks = append(ensureCRDsTasks, sh_task.NewTask(task.ModuleEnsureCRDs).
```

Comment:
seems like we add ensure-crds task for every module. It could be a bit expensive, to scan filesystem every time for every module. 
I assume, we have to add a flag like `CRDsExist: bool` to the BasicModule and fill it somewhere in the loader:
1. https://github.com/flant/addon-operator/blob/main/pkg/module_manager/loader/fs/fs.go#L63
2. https://github.com/deckhouse/deckhouse/blob/main/deckhouse-controller/pkg/controller/loader.go#L106

@miklezzzz what do you think?

Reply:
yep, makes sense, we would need:
1) add a flag to the basic module definition about having/not having CRDs to apply
2) set this flag somewhere around modulemanager's Init() call, I suppose module loader has to be updated to support CRD discovery

Reply:
Also, I personally think we should apply CRD's only on Startup, but I might be wrong..

Reply:
if someone is messing up with CRDs when operator is running it shouldn't be out problem, should it? @yalosev

Reply:
Does enabling a module generate an onStartup task?

Reply:
DONE

Reply:
awesome, let's add something like ModuleHasCRDs method to the module manager and ask it before creating a moduleEnsureCRD task, it will result in having fewer tasks and less info when debugging/logging

Reply:
DONE

Reply:
The only thing I think is that this excess overloads the logic. But it doesn't give much profit.